README.md

```diff
@@ -17,22 +17,36 @@ deploy changes with:
 Editing
 ------------------------------
 
-### How to set up the project
+### Prerequisites
 
-visit: https://github.com/lord/slate
+You're going to need:
+
+ - **Linux or macOS** — Windows may work, but is unsupported.
+ - **Ruby, version 2.3.1 or newer**
+ - **Bundler** — If Ruby is already installed, but the `bundle` command doesn't work, just run `gem install bundler` in a terminal.
 
 ### Getting Set Up
 
-run locally:
+1. Fork this repository on GitHub.
+2. Clone *your forked repository* (not our original one) to your hard drive with `git clone https://github.com/YOURUSERNAME/slate.git`
+3. `cd slate`
+4. Initialize and start Slate. You can either do this locally, or with Vagrant:
 
 ```shell
-./run.sh
+# either run this to run locally
+bundle install
+bundle exec middleman server
+
+# OR run this to run with vagrant
+vagrant up
 ```
 
-You can now see the docs at http://localhost:4567. 
+You can now see the docs at http://localhost:4567. Whoa! That was fast!
 
-deploy changes with:
+Now that Slate is all set up on your machine, you'll probably want to learn more about [editing Slate markdown](https://github.com/lord/slate/wiki/Markdown-Syntax), or [how to publish your docs](https://github.com/lord/slate/wiki/Deploying-Slate).
 
-```shell 
-./deploy.sh
```

Comment:
You remove here deployment instructions, but this is important to have them. Or isn't it ?

Reply:
It is already at the top of readme.md file. This was changed by accident.